app/internal/logger_customizer.py

```diff
@@ -0,0 +1,110 @@
+import json
+import sys
+from typing import Union
+
+from pathlib import Path
+from loguru import logger, _Logger as Logger
+
+
+class LoggerConfigError(Exception):
+    pass
+
+
+class LoggerCustomizer:
+
```

Comment:
Redundant blank line?

Reply:
Which one? I know PEP8 says 2 blank lines between classes.

Reply:
I meant to line 14 (inside the class).